### PR TITLE
Fix test case about eye

### PR DESCRIPTION
**Pull request by liufengwei0103**

### CODE REVIEW

**Review by MARD1NO — APPROVED** [review]

**Review by Flowingsun007 — APPROVED** [review]

File: python/oneflow/test/modules/test_eye.py [hunk 1]
```diff
@@ -59,14 +59,12 @@ def test_eye(test_case):
         for arg in GenArgList(arg_dict):
             arg[0](test_case, *arg[1:])
 
-    @autotest(check_graph=False)
+    @autotest(auto_backward=False, check_graph=False)
```
Comment (BBuf) [inline python/oneflow/test/modules/test_eye.py:62]: 考虑到check_graph可以改成：

```python
@autotest(check_graph=True)
    def test_eye_with_random_data(test_case):
        n = random(low=1, high=5).to(int)
        m = random(low=1, high=5).to(int)
        x = torch.eye(n=n, m=m)
        x.oneflow.requires_grad = True
        x.pytorch.requires_grad = True
        device = random_device()
        x.to(device)
        return x
```

然后0-dim tensor可以单独加一个

Comment (liufengwei0103) [reply python/oneflow/test/modules/test_eye.py:62]: > 考虑到check_graph可以改成：
> 
> ```python
> @autotest(check_graph=True)
>     def test_eye_with_random_data(test_case):
>         n = random(low=1, high=5).to(int)
>         m = random(low=1, high=5).to(int)
>         x = torch.eye(n=n, m=m)
>         x.oneflow.requires_grad = True
>         x.pytorch.requires_grad = True
>         device = random_device()
>         x.to(device)
>         return x
> ```
> 
> 然后0-dim tensor可以单独加一个

OK
